updated contents in ML and in padgett use cases

silviamazzoni opened the pull request:
## Overview

Updated contents in ML and in `padgett` use cases.

## Related

- updates doc from #35

## Changes

- **fixes** new and related docs to be consistent

## Testing

…

## UI

See #40, which has one mistake, which is fixed by b5ad6e9.

---

_Edited by admin._

silviamazzoni committed:
updated contents in ML and in padgett use cases

wesleyboar committed:
chore: delete user config files

wesleyboar committed:
Revert "chore: delete user config files"

This reverts commit dfccb5cc083568ef305957ef70b6c4a1848377f8.

wesleyboar reviewed (commented):
I noticed some `.vs` config dir files got added. I assume these are from and for your local editing environment. I made a "sub-PR" where I delete them (#39), and I updated `main` branch to ignore some such directories.

_**Update:** PR closed. I'll deal with these after merge. It's easier than teaching a solution I'm not skilled at._

wesleyboar commented on user-guide/docs/usecases/vantassel_and_zhang/usecase.md:
```diff
@@ -13,7 +13,28 @@ of machine learning approaches (as there are many), nor in its coverage of the s
+
+The following DesignSafe resources are leveraged in this example: 
+
+[Geospatial data analysis and Visualization on DS - QGIS](https://www.designsafe-ci.org/rw/workspace/#!/qgis-duvd-3.16.3u2){target=_blank}<br/>
+[Jupyter notebooks on DS Jupyterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){target=_blank}
```
<details><summary>🤔 Why new lines and <code>&lt;br /&gt;</code> instead of list items?</summary>

```suggestion
- [Geospatial data analysis and Visualization on DS - QGIS](https://www.designsafe-ci.org/rw/workspace/#!/qgis-duvd-3.16.3u2){target=_blank}
- [Jupyter notebooks on DS Jupyterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){target=_blank}
```

</details>

wesleyboar commented on user-guide/docs/usecases/vantassel_and_zhang/usecase.md:
```diff
@@ -58,14 +79,24 @@ machine learning problems.
 
-[Linear Regression Example](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression){target=_blank}
+Open Jupyter Notebook dirctly:
+- Linear Regression, Real Estate Data Set: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression/0_real_estate.ipynb) 
```
<details><summary>🧐 I believe a new line is required before a list. MkDocs markdown is less forgiving than Github's.</summary>

```suggestion

- Linear Regression, Real Estate Data Set: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression/0_real_estate.ipynb) 
```

</details>

wesleyboar commented on user-guide/docs/usecases/vantassel_and_zhang/usecase.md:
```diff
@@ -58,14 +79,24 @@ machine learning problems.
+- Linear Regression, Real Estate Data Set: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression/0_real_estate.ipynb) 
+- Linear Regression, Real Estate Data Set - Learning Curves: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression/1_real_estate_learning_curves.ipynb) 
+- Linear Regression, Real Estate Data Set - Regularization: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression/2_real_estate_regularization.ipynb) 
+or
+View in the Data Depot:
```
<details><summary>🧐 I believe new lines are required to break these onto a new line outside the list.</summary>

```suggestion

or
View in the Data Depot:
```

</details>

wesleyboar replied:
Suggestion tweaked after testing.

silviamazzoni committed:
fixed a link to a notebook

wesleyboar commented on user-guide/docs/usecases/vantassel_and_zhang/usecase.md:
```diff
@@ -58,14 +79,24 @@ machine learning problems.
+Open Jupyter Notebook dirctly:
+- Random forest classification -- Multi-Class: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/1_random_forest/RandomForestClassification_MultiClass.ipynb) 
+or
+View in the Data Depot:
```
<details><summary>🧐 I believe new lines (or maybe a <code>&lt;br/&gt;</code>) are required to achieve what you expect.</summary>

```suggestion
Open Jupyter Notebook dirctly:

- Random forest classification -- Multi-Class: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/1_random_forest/RandomForestClassification_MultiClass.ipynb)

or
View in the Data Depot:
```

</details>

wesleyboar replied:
Suggestion tweaked after testing.

wesleyboar commented on user-guide/docs/usecases/vantassel_and_zhang/usecase.md:
```diff
@@ -74,6 +105,10 @@ Artificial neural networks (ANNs), usually simply called neural networks (NNs),
+Open Jupyter Notebook dirctly:
+- Neural Networks: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/2_artificial_neural_networks/ANNClassification_MultiClass.ipynb) 
+or
+View in the Data Depot:
```
<details><summary>🧐 I believe new lines (or maybe a <code>&lt;br/&gt;</code>) are required to achieve what you expect.</summary>

```suggestions
Open Jupyter Notebook dirctly:

- Neural Networks: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/2_artificial_neural_networks/ANNClassification_MultiClass.ipynb)

or
View in the Data Depot:
```

wesleyboar replied:
Suggestion tweaked after testing.

silviamazzoni committed:
Merge branch 'silvia/use-cases' of https://github.com/DesignSafe-CI/DS-User-Guide into silvia/use-cases

wesleyboar commented on user-guide/docs/usecases/vantassel_and_zhang/usecase.md:
```diff
@@ -89,4 +124,9 @@ resources a replica of the main notebook is provided that can run across multipl
+- Convolutional Neural Networks, MNIST Dataset: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/3_convolutional_neural_networks/0_mnist.ipynb) 
+- Convolutional Neural Networks, MNIST Dataset - Parallel: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/3_convolutional_neural_networks/1_mnist_parallel.ipynb) 
+or
+View in the Data Depot:
```
<details><summary>🧐 I believe new lines (or maybe a <code>&lt;br/&gt;</code>) are required to achieve what you expect.</summary>

```suggestion
Open Jupyter Notebook dirctly:

- Convolutional Neural Networks, MNIST Dataset: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/3_convolutional_neural_networks/0_mnist.ipynb) 
- Convolutional Neural Networks, MNIST Dataset - Parallel: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/3_convolutional_neural_networks/1_mnist_parallel.ipynb)

or
View in the Data Depot:
```

</details>

wesleyboar replied:
Suggestion tweaked after testing.

wesleyboar commented on user-guide/docs/usecases/padgett/usecase_JN_viz.md:
```diff
@@ -1,48 +1,76 @@
+#### DesignSafe Resources
+The following DesignSafe resources were used in developing this use case.
+
+* [Jupyter Notebook on DesignSafe](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){:target="_blank"} <br/> 
```
<details><summary>🤔 What is the intent of the <code>&lt;br/&gt;</code>? The end of the list item and the list should already achieve a new line.</summary>

```suggestion
* [Jupyter Notebook on DesignSafe](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){:target="_blank"}
```

</details>

wesleyboar replied:
| with `<br/>`<br><sup>has no effect</sup> | sans `<br/>`<br><sup>renders the same</sup> |
| - | - |
| <img width="1200" alt="usecase_JN_viz with br (has no effect)" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/e85e74c6-fce8-4395-af57-ee37748ef503"> | <img width="1200" alt="usecase_JN_viz sans br (renders the same)" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/6398151d-5693-4fe8-9e79-8bd166adce40"> |

silviamazzoni commented:
oh, great. I tried doing it a different way: in github in the web i asked it to open directly in Visual studio, and it opened a new version of it.....
what should I do?

wesleyboar commented on user-guide/docs/usecases/padgett/usecase_JN_viz.md:
I see you changed image paths. My [testing](https://github.com/DesignSafe-CI/DS-User-Guide?tab=readme-ov-file#testing) shows:
- Paths like `./`. work on GitHub, **but not** when deployed.
- Paths like `../` work when directly viewing page, **but not** when loading doc on compiled page.
- Paths like `/user-guide/usecases/…/` work when directly viewing page **and** loading doc on compiled page.

Reference:
- https://github.com/DesignSafe-CI/DS-User-Guide/pull/7
- https://github.com/DesignSafe-CI/DS-User-Guide/pull/35#pullrequestreview-2103244311

wesleyboar replied:
I was wrong. I am confused now. I will learn image paths another day.

Thanks for the fix.

wesleyboar reviewed (commented):
Thanks for the fixes. Consistency for the win.

I have questions on syntax. I worry about image paths.

silviamazzoni commented:
i see..... the other times I was using VS code, not just VS....

wesleyboar committed:
Merge branch 'main' into silvia/use-cases

wesleyboar commented:
I'll manage the `.vs` dirs. You should not need to worry about 'em later also. Git should ignore them after I'm done.

wesleyboar commented on user-guide/docs/usecases/vantassel_and_zhang/usecase.md:
```diff
@@ -58,14 +79,24 @@ machine learning problems.
+- Linear Regression, Real Estate Data Set - Regularization: [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/CommunityData/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression/2_real_estate_regularization.ipynb) 
+or
+View in the Data Depot:
+(https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression){target=_blank}
```
```suggestion
[Linear Regression Example](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/0_linear_regression){target=_blank}
```

wesleyboar reviewed (commented):
1. I was wrong about image paths.
2. I have tested and taken screenshots.

<details><summary>"before & after" screenshots</summary>

| before | after |
| - | - |
| ![vantassel_and_zhang BEFORE](https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/a06de320-7dac-4d8e-abbd-72808032c11b) | ![vantassel_and_zhang AFTER](https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/b551f44f-75ae-494c-8309-86b178997cbf) |

</details>

wesleyboar committed:
silvia/use-cases: suggestions in bulk (#40)

(in case contributor is unaware how to piecemeal or bulk add suggestions through github)

wesleyboar commented on user-guide/docs/usecases/vantassel_and_zhang/usecase.md:
```diff
@@ -66,6 +95,11 @@ Random forests or random decision forests is an ensemble learning method for cla
 
 ![image_of_5_with_label](img/1_random_forest.png)
 
+Open Jupyter Notebook dirctly:
```
```suggestion
Open Jupyter Notebook directly:

```

wesleyboar reviewed (commented):
Resolved several concerns via merge of #40.

wesleyboar committed:
fix: typo and missing new line
